spec/models/room_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Room, type: :model do
   describe 'relationships' do
     it {should belong_to :hotel}
+    it {should have_many :guest_rooms}
+    it {should have_many(:guests).through(:guest_rooms)}
   end
 end
```

Comment:
Having moved the `count` method to the model, you would have also needed to test it here.